projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/serverRequests.ts

```diff
@@ -75,7 +80,11 @@ export async function jobRequest(endPoint: string): Promise<void> {
         body: JSON.stringify(getJobDataJsonObject()),
         method: 'POST'
       });
-      if (!data['error']) alert(data['message']);
+      if (!data['error'])
+        return {
+          message: data['message'],
+          status: true
```

Comment:
what does status being true mean? It's a bit ambiguous. 
Let's be explicit and name it `isSuccessful`  or `success` ?

Reply:
changed to isSuccessful